### ISSUE TITLE
Check that eth node is synced before starting service

### ISSUE THREAD

**Issue by karlb** [raiden-network/raiden-services#873]
Check the `eth.syncing` flag during startup. Otherwise the service will give bad results due to an outdated eth node.

- [ ] Check in PFS
- [ ] Check in MS

**Comment by karlb** [comment]
Duplicate of https://github.com/raiden-network/raiden-services/issues/655
